scripts/jtsync/README.md

```diff
@@ -0,0 +1,58 @@
+# jtsync - synchronize status from jenkins to trello
+    Usage: jtsync --ci SERVICE (--matrix|--job) JOB_STATUS
+            --ci SERVICE                 Which ci is used (suse or opensuse)
+            --matrix NAME,PROJECT        Set status of a matrix job
+            --job NAME                   Set status of a normal job
+
+Matrix job cards are selected in this form:
+
+    $ jtsync --ci suse --matrix cloud-trackupstream,Devel:Cloud:7:Staging
+
+The script never returns anything other than `0` to make sure jenkins runs are not interruped
+by the script.
+
+
+## Run the script on a developer machine
+Install `bundler` before continue:
+
+    cd <path to jtsync directory>
+    bundler install --path=~/.bundle-jtsync
+    bundle exec jtsync.rb
+    ...
+
+## How to test:
+In general there is the `test_jtsync` script to run an manual test run. You need
+to create credentials for the trello API (you can create credentials [here](https://trello.com/app-key))
+Place them in your local `.netrc` file:
+
+    machine api.trello.com
+      login <developer token>
+      password <member token>
+
+After that create a new trello board or clone [https://trello.com/b/fL7fv67z/jtsync](https://trello.com/b/fL7fv67z/jtsync).
+
+Replace the _hardcoded_ __board id__ in `jtsync.rb` to your test __board id__.
```

Comment:
Also it would be better to allow the board id to be specified via the CLI.  That way `test_jtsync.rb` could default to using the existing test board.  Currently if someone runs `test_jtsync.rb` without reading these instructions, it would corrupt the status of the real Jenkins Green board.